ironfish/src/mining/director.ts

```diff
@@ -331,15 +331,14 @@ export class MiningDirector {
       throw Error('No miner account found to construct the transaction')
     }
 
-    const blockTransactions = []
+    const blockTransactions: Transaction[] = []
 
     // Fetch all transactions for the block
     for await (const transaction of this.getTransactions()) {
       if (blockTransactions.length >= MAX_TRANSACTIONS_PER_BLOCK) {
         break
       }
 
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
       blockTransactions.push(transaction)
```

Comment:
Nice. I just saw this and fixed it in another PR. I can't believe this was using any here...